instances/atlas/parcellationEntityVersion/WHSSD_v1-01/WHSSD_v1-01_cingulateCortexArea2.jsonld

```diff
@@ -22,7 +22,11 @@
         ],
         "visualizedIn": null
     },
-    "hasParent": null,
+    "hasParent": [
+        {
+            "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/WHSSD_cerebralCortexIncludingTheNeocortexAndTheHippocampus"
```

Comment:
is there another cerebralCortex ? if not please shorten this id to the necessary part